modules/docs/lib/widgetUtils.tsx

```diff
@@ -80,18 +93,61 @@ export interface SymbolDialogProps {
 
 export const SymbolDialog = ({value}: SymbolDialogProps) => {
   const [symbol, setSymbol] = React.useState<types.ExportedSymbol | undefined>(undefined);
+  const {breadcrumbsList, updateBreadcrumbs} = React.useContext(SymbolDocBreadcrumbsContext)!;
+  const nestedContext = React.useContext(NestedContext);
+
   const model = useDialogModel({
     onShow() {
       setSymbol(docs.find(d => value.name === d.name) || undefined);
     },
+    onHide() {
+      // Reset breadcrumbs when the dialog is closed
+      updateBreadcrumbs([]);
+    },
   });
 
+  const handleTargetClick = (e: React.MouseEvent) => {
+    e.preventDefault();
+
+    updateBreadcrumbs(breadcrumbsList.concat(value.name));
+  };
+
+  const handleBreadcrumbClick = (e: React.MouseEvent, index: number) => {
+    e.preventDefault();
+    if (
+      e.currentTarget.textContent &&
+      breadcrumbsList.length &&
+      breadcrumbsList.indexOf(e.currentTarget.textContent) > -1
+    ) {
+      // Slice breadcrumbs list up to but not including the current item clicked
+      updateBreadcrumbs(breadcrumbsList.slice(0, index + 1));
+    }
+  };
+
+  // If we're not within a nested context, we render a Hyperlink. This help that dialog know it's target.
+  if (nestedContext) {
+    return (
+      <ButtonHyperLink
+        style={{
+          border: 'none',
+          background: 'transparent',
+          fontSize: 'inherit',
+          fontFamily: 'inherit',
+        }}
+        onClick={handleTargetClick}
+      >
+        {' '}
```

Comment:
Is this space necessary? It isn't in the `Dialog.Target` so my guess is this will add an extra space or collapse depending on the whitespace CSS property.
```suggestion
```

Reply:
prettier added this, I'll look into it

Reply:
I'm sure there was a space somewhere and prettier forced the space when reformatting